uint32[] port_handles
string[] marker_names
float64[] timestamps
uint64[] framenumbers
geometry_msgs/Pose[] tracking
float64[] quality